lib/Composer.php

```diff
@@ -234,8 +234,8 @@ protected static function getOptions(Event $event)
     {
         $options = array_merge(static::$options, $event->getComposer()->getPackage()->getExtra());
 
-        $options['symfony-assets-install'] = getenv('SYMFONY_ASSETS_INSTALL') ?: $options['symfony-assets-install'];
-        $options['symfony-cache-warmup'] = getenv('SYMFONY_CACHE_WARMUP') ?: $options['symfony-cache-warmup'];
+        $options['symfony-assets-install'] = $_SERVER['SYMFONY_ASSETS_INSTALL'] ?: $options['symfony-assets-install'];
+        $options['symfony-cache-warmup'] = $_SERVER['SYMFONY_CACHE_WARMUP'] ?: $options['symfony-cache-warmup'];
```

Comment:
```suggestion
        $options['symfony-assets-install'] = $_SERVER['SYMFONY_ASSETS_INSTALL'] ?? $options['symfony-assets-install'];
        $options['symfony-cache-warmup'] = $_SERVER['SYMFONY_CACHE_WARMUP'] ?? $options['symfony-cache-warmup'];
```

Reply:
Have fixed it with `!empty($_SERVER['SYMFONY_ASSETS_INSTALL'])` - is also easier to read.